dev/com.ibm.ws.jaxrs.2.0.common/bnd.bnd

```diff
@@ -164,10 +164,10 @@ instrument.classesExcludes: com/ibm/ws/jaxrs20/internal/resources/*.class, \
 	org.apache.cxf.cxf-tools-common;strategy=exact;version=3.1.18,\
 	org.apache.cxf.cxf-tools-wadlto-jaxrs;strategy=exact;version=3.1.18,\
 	org.apache.cxf.cxf-rt-rs-service-description;strategy=exact;version=3.1.18,\
-	com.ibm.ws.org.codehaus.jackson:jackson-core-asl;version=1.6.2, \
-	com.ibm.ws.org.codehaus.jackson:jackson-jaxrs;version=1.6.2, \
-	com.ibm.ws.org.codehaus.jackson:jackson-mapper-asl;version=1.6.2, \
-	com.ibm.ws.org.codehaus.jackson:jackson-xc;version=1.6.2, \
+	com.ibm.ws.org.codehaus.jackson:jackson-core-asl;version=1.6.2.1, \
+	com.ibm.ws.org.codehaus.jackson:jackson-jaxrs;version=1.6.2.1, \
+	com.ibm.ws.org.codehaus.jackson:jackson-mapper-asl;version=1.6.2.1, \
+	com.ibm.ws.org.codehaus.jackson:jackson-xc;version=1.6.2.1, \
```

Comment:
These four lines can all be replaced with `com.ibm.ws.org.codehaus.jackson.1.6.2;version=latest,\` (after undoing the change to the Jackson bundle's symbolic name).  This could speed up build times since it can pull the dependencies from the already-built Jackson bundle rather than downloading the dependencies from DHE.